common/model/color-selections.ts

```diff
@@ -0,0 +1,8 @@
+export const ColorSelections = {
+  Turquoise: 'turquoise',
+  Red: 'red',
+  Green: 'green',
+  Purple: 'purple',
+} as const;
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions

Reply:
Nice.

Reply:
Why not use a union type of the string values though?

Reply:
That's a good question. I think the original intention was that we could refer to this `ColorSelections` variable within the code itself, so that the type and variable would stay in sync. Although a repo-wide search suggests we're not using it as  a variable anywhere. I guess we still might/could though.